src/error.rs

```diff
@@ -0,0 +1,114 @@
+use cosmwasm_std::{
```

Comment:
Most of these errors can be removed, I just started with a base one and worked off of it.